src/api/fusion-api.ts

```diff
@@ -39,6 +47,12 @@ export class FusionApi {
         return this.quoterApi.getQuote(params)
     }
 
+    getActiveOrders(
+        request: ActiveOrdersRequest
```

Comment:
1. Rename `request` to `params`
2. Assign `ActiveOrdersRequest.new()` as a default value

Reply:
1. done
2. Do we really need this, is there any case when user calls `FusionApi` class directly instead of `FusionSDK`

Reply:
Someone can use api separate from sdk, for example, resolvers. They don't need to place orders, they should just fetch users orders

Reply:
Ok, that make sense right now